virttest/utils_time.py

```diff
@@ -8,15 +8,17 @@
 from virttest import utils_test
 from virttest.compat_52lts import decode_to_text
 
+# command `grep --color` may have alias name `grep` in some systems,
+# so get explicit command 'grep' with path
+grep_binary = decode_to_text(process.system_output("which grep|grep -v alias", shell=True))
```

Comment:
This should help here,

```python
from avocado.utils import path
path.find_command("grep")
```

Reply:
@balamuruhans perfect!, will use it, thanks for the suggestion...